From 048740ed1f2523fb8d6afc7c906ddfbe1d3705e2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jakab=20Krist=C3=B3f?= <jakab.kristof@balasys.hu>
Date: Thu, 6 Oct 2016 16:13:37 +0200
Subject: [PATCH 3/4] Only build amd64-generic and amd64-debug

---
 debian.xenial/rules.d/amd64.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian.xenial/rules.d/amd64.mk b/debian.xenial/rules.d/amd64.mk
index e97df86..8647a37 100644
--- a/debian.xenial/rules.d/amd64.mk
+++ b/debian.xenial/rules.d/amd64.mk
@@ -2,7 +2,7 @@ human_arch	= 64 bit x86
 build_arch	= x86_64
 header_arch	= $(build_arch)
 defconfig	= defconfig
-flavours	= generic lowlatency
+flavours	= generic debug
 build_image	= bzImage
 kernel_file	= arch/$(build_arch)/boot/bzImage
 install_file	= vmlinuz
-- 
1.9.1

